src/core_plugins/elasticsearch/index.js

```diff
@@ -15,8 +16,6 @@ const DEFAULT_REQUEST_HEADERS = [ 'authorization' ];
 
 module.exports = function ({ Plugin }) {
   return new Plugin({
-    require: ['kibana'],
```

Comment:
I also love that this makes the mappings static, so they can't be changed at runtime. It would definitely be easiest to add these to `uiExports`... is it too far of a stretch to call the mappings a `uiExport`?